src/astro/databases/google/bigquery.py

```diff
@@ -236,3 +257,177 @@ def gs_to_bigquery(
         self.hook.insert_job(
             configuration=job_config,
         )
+
+    def s3_to_bigquery(
```

Comment:
This way we'd be consistent with the current naming conventions:
```suggestion
    def load_s3_file_to_table(
```